internal/cli/cmd/sysdump.go

```diff
@@ -74,6 +74,9 @@ func initSysdumpFlags(cmd *cobra.Command, options *sysdump.Options, optionPrefix
 	cmd.Flags().StringVar(&options.CiliumEnvoyLabelSelector,
 		optionPrefix+"cilium-envoy-label-selector", sysdump.DefaultCiliumEnvoyLabelSelector,
 		"The labels used to target Cilium Envoy pods")
+	cmd.Flags().StringVar(&options.CiliumHelmChartName,
```

Comment:
This `CiliumHelmChartName` isn't used as the chart name, but rather the release name. So this flag and the `sysdump.DefaultCiliumHelmChart` name should instead be `..ReleaseName..``